.changeset/forty-masks-serve.md

```diff
@@ -0,0 +1,13 @@
+---
+'@builder.io/react': patch
+'@builder.io/sdk-angular': patch
+'@builder.io/sdk-react-nextjs': patch
+'@builder.io/sdk-qwik': patch
+'@builder.io/sdk-react': patch
+'@builder.io/sdk-react-native': patch
```

Comment:
oh, for react native we use this override: https://github.com/sidmohanty11/builder/blob/general-support-vc/packages/sdks/overrides/react-native/src/blocks/image/image.tsx/#L29 can you please check if this supports title? else we can remove this addition from react-native

Reply:
Done